src/plugins/plot/inspector/PlotOptionsItem.vue

```diff
@@ -79,14 +82,8 @@
             {{ markerOptionsDisplayText }}
           </div>
         </li>
-        <li class="grid-row" role="row">
-          <div
-            class="grid-cell label"
-            title="Display markers visually denoting points in alarm."
-            role="cell"
-          >
-            Alarm Markers
-          </div>
+        <li class="grid-row" role="row" title="Display markers visually denoting points in alarm.">
```

Comment:
moved title to the list instead of the Div. This allows us to chain on the title attribute in the test